nginx.conf

```diff
@@ -113,6 +113,9 @@
 
     # Alertmanager
     location /alertmanager {
+      satisfy  any;
+      allow 127.0.0.1;
+      deny all;
```

Comment:
Why do we deny request to `/alertmanager` for all except localhost?

Reply:
Yes. This deny rule has no effect here. I have removed it.
New datasource plugin has to have access to  /alertmanager without authentication.

Reply:
Why previously it was available for all users? @AlekSi maybe you know?

Reply:
It's not available for all users. It's still covered by 
![Screenshot_20200918_135215](https://user-images.githubusercontent.com/25184101/93589898-3d896500-f9b6-11ea-864b-0d6f74cfca7e.png)

Reply:
ok, for admin users. it's not available now. because of this changes.

Reply:
> New datasource plugin has to have access to /alertmanager without authentication.

Why? Nothing in the ticket mentions that

Reply:
I have added information about required changes into the ticket.

Reply:
But I asked _why_. I see what you did, but I don't understand why.

Reply:
It's required to set an admin credential for access to location /alertmanager.
How we can be sure that default admin/admin wouldn't be changed?

Reply:
We know for sure that admin/admin will be changed, that's why we use cookies for authentication and Grafana roles for authorisation. That nginx configuration bypasses that, allowing everyone to access this data. We should not do that.